builds/e2e/templates/connectivity-deploy.yaml

```diff
@@ -63,6 +63,7 @@ steps:
           -metricsEndpointsCSV "${{ parameters['metricsCollector.metricsEndpointsCSV'] }}" \
           -metricsScrapeFrequencyInSecs "${{ parameters['metricsCollector.scrapeFrequencyInSecs'] }}" \
           -metricsUploadTarget "${{ parameters['metricsCollector.uploadTarget'] }}" \
+          -deploymentFileName "$(deploymentFileName)" \
```

Comment:
why this is not passing from parameters? for example "edgelet.artifact.name" or "testDuration"

Reply:
When I did the `${{ parameters['edgelet.artifact.name'] }}` it gives me an empty string. which was super weird given that in connectivity.yaml I had `deploymentFileName: $(deploymentFileName)`